system/handlers/admin/DataManager.cfc

```diff
@@ -194,11 +194,24 @@
 
 			_checkPermission( argumentCollection=arguments, key="read", object=objectName );
 
+			var filterByField = rc.filterByField ?: "";
+			var extraFilters  = [];
+
+			loop list=filterByField index="indexField"{
```

Comment:
`indexField` is not `var'd` here and I would avoid looping a list - especially using cfloop syntax. Prefer this:

```
var filterByFields = ListToArray( rc.filterByField ?: "" );
for( var filterByField in filterByFields ) {
    // ...
}
```